doc/content/documentation/repl.md

```diff
@@ -3,53 +3,64 @@ title = "REPL"
 weight = 18
 +++
 
-Phel provides a interactive prompt where you can type Phel expressions and see the results. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
+Phel comes with an interactive prompt. The prompt accepts Phel expressions and directly returns the result. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
 
-The REPL can be started with the following command:
+The REPL is started with the following command:
 ```bash
 ./vendor/bin/phel repl
 ```
 
-Afterwards you can type an expression and see the result intermediately
+Afterwards any Phel expression can be typed in.
 
 ```bash
 Welcome to the Phel Repl
 Type "exit" or press Ctrl-D to exit.
->>> (* 6 7)
+phel:1> (* 6 7)
 42
->>>
+phel:2>
+```
+
+The prompt also accepts multiline expressions:
+```bash
+Welcome to the Phel Repl
+Type "exit" or press Ctrl-D to exit.
```

Comment:
This is out of context.
Why "Ctrl + D" instead of "Ctrl + C"?

Reply:
Because it's what you see when entering the REPL 😄
<img width="416" alt="Screenshot 2021-01-30 at 23 33 13" src="https://user-images.githubusercontent.com/5256287/106369650-86cdc600-6353-11eb-9aad-5dd814549b62.png">

Reply:
Uh... There is a difference between `Ctrl+D` and `Ctrl+C` on how the program is terminated. Maybe it is better your google it. I probably explain it wrong.

Reply:
https://dev.to/konkit/obvious-and-not-so-obvious-bash-zsh-terminal-shortcuts-3480#:~:text=Ctrl%2DC%20and%20Ctrl%2DD,D%20exits%20the%20REPL%20itself. :)